### PR TITLE
Fix dockerfile, update makefile

### PR DESCRIPTION
**Pull request by fkrause98**
## Changes:
- Overall, this changes make sure you can run make setup + make run on a fresh clone, 
  or build the docker image.
- Fix dockerfile by adding git submodule init.
- Some doc updates.
- Makefile target updates.
- Update git submodule for new wasm changes on lambdaworks

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM elixir:1.14.5-otp-25 as builder
 
-RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain=1.70
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain=1.69
```
Comment (d-roak) [inline Dockerfile:3]: Why downgrade the toolchain? 1.70 works fine

File: Makefile [hunk 2]
```diff
@@ -1,12 +1,22 @@
-.PHONY: run setup deps-get db stop-db setup-wasm-prover
+.PHONY: run setup deps-get db stop-db setup-wasm-prover setup-assets start-db create-db
 
 run:
 	iex -S mix phx.server
 
-setup: deps-get db
+setup: submodule deps-get start-db setup-assets create-db
```
Comment (klaus993) [inline Makefile:6]: @fkrause98 should this target (`setup`) also install wasm-pack (`cargo install wasm-pack`)?
I cloned this repo and ran `make setup` and it failed with `/bin/sh: wasm-pack: command not found`

**Review by klaus993 — APPROVED** [review]
Besides my comment about wasm-pack, looks great! I tested it locally, pointing it to our internal Juno, and it works just fine.

File: Makefile (hunk 2 above)
Comment (fkrause98) [reply Makefile:6]: That's a good idea, I'll update the makefile

**Comment by fkrause98** [comment]
Closing as we won't be using the prove for now.

**Pull request closed without merging**